generators/generator-base.js

```diff
@@ -2267,6 +2267,153 @@ templates: ${JSON.stringify(existingTemplates, null, 2)}`;
         );
   }
 
+  /**
+   * write the given files using provided options.
+   *
+   * @param {object} options
+   * @param {object} options.files - files to write
+   * @param {object} [options.context] - context for templates
+   * @param {string|string[]} [options.rootTemplatesPath] - path(s) to look for templates.
+   *        Single absolute path or relative path(s) between the templates folder and template path.
+   * @return {string[]}
+   */
+  async writeFiles(options) {
+    const { files, context = this } = options;
```

Comment:
is it possible to create a dedicated object for this?
example:
`utils/generation/model/templates-context.js`

Reply:
This is just a parameter grouping, no need.